core/src/main/scala/kafka/server/ClientRequestQuotaManager.scala

```diff
@@ -37,18 +37,18 @@ class ClientRequestQuotaManager(private val config: ClientQuotaManagerConfig,
   }
 
   def maybeRecordAndThrottle(request: RequestChannel.Request, sendResponseCallback: Int => Unit): Unit = {
-    if (request.apiRemoteCompleteTimeNanos == -1) {
+    if (request.summary.apiRemoteCompleteTimeNanos == -1) {
```

Comment:
All the places that only need the summary should be changed to receive the summary.

Reply:
Ok.